packages/core/chrome/core-chrome-browser/src/project_navigation.ts

```diff
@@ -71,13 +78,16 @@ export interface ChromeProjectNavigation {
    * The navigation tree representation of the side bar navigation.
    */
   navigationTree: ChromeProjectNavigationNode[];
+  /**
+   * The flattened navigation tree representation of the side bar navigation.
+   * This is optional, if not provided it will be computed from the navigationTree
+   */
+  navigationTreeFlattened?: Record<string, ChromeProjectNavigationNode>;
 }
 
 /** @public */
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface SideNavCompProps {
-  // TODO: provide the Chrome state to the component through props
-  // e.g. "navTree", "activeRoute", "recentItems"...
+  activeNodes: ChromeProjectNavigationNode[][];
```

Comment:
Yes the only way it is not showing in the side nav is because it does not have a `title` or it is an empty string `""`. I can add a flag but it would follow that rule